packages/pn-personafisica-webapp/src/component/Contacts/SpecialContacts.tsx

```diff
@@ -284,21 +322,21 @@ const SpecialContacts = ({ recipientId, legalAddresses, courtesyAddresses }: Pro
         <form style={{ margin: '20px 0' }} onSubmit={formik.handleSubmit}>
           <Grid container direction="row" spacing={2} alignItems="flex">
             <Grid item lg xs={12}>
-              <CustomDropdown
+              <Autocomplete
                 id="sender"
-                label={`${t('special-contacts.sender', { ns: 'recapiti' })}*`}
-                name="sender"
-                value={formik.values.sender}
-                onChange={senderChangeHandler}
-                fullWidth
                 size="small"
-              >
-                {parties.map((party) => (
-                  <MenuItem key={party.id} value={party.id}>
-                    <DropDownPartyMenuItem name={party.name} />
-                  </MenuItem>
-                ))}
-              </CustomDropdown>
+                options={parties}
+                fullWidth
+                autoComplete
+                getOptionLabel={getOptionLabel}
+                isOptionEqualToValue={(option, value) => option.name === value.name}
+                onChange={senderChangeHandler}
+                inputValue={inputValue}
+                onInputChange={(_event, newInputValue) => handleChangeInput(newInputValue)}
+                filterOptions={handleFilterAutocomplete}
+                renderOption={renderOption}
+                renderInput={(params) => <TextField {...params} name="sender" label={ t('special-contacts.sender')} />}
```

Comment:
Translation is not rendered properly. Add
, { ns: 'recapiti' }
after t() function

Reply:
do it